ext/vscode/.vscode/launch.json

```diff
@@ -19,23 +19,30 @@
                 "DEBUGTELEMETRY": "true"
             },
             "outFiles": [
-                "${workspaceFolder}/dist/**/*.js"
-            ]
+                "${workspaceFolder}/dist/**/*.js",
```

Comment:
Changes in this file are unrelated to the bug fix, but without them I could not get the debugger to work with the VS Code ext. Not sure what magic Brandon was using.. I am willing to endure his wrath for these when he comes back, but I need them for now.